LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I usually use a more permissive license, such as BSD 2-Clause or MIT, though this is obviously fine. Looks like Split-Seq is MIT though, so you might consider using a compatible license, just for simplicity sake.

Reply:
Updated (yet to push).